src/controllers/delegate.ts

```diff
@@ -31,8 +31,8 @@ export function DelegatePost(
     return;
   }
 
```

Comment:
OPT: we should probably enforce check on `req.body.duration` as having a hidden default duration server side will likely lead to bugs not easy to troubleshoot

Reply:
done